File: Objects_Practice/Car.java
package Objects_Practice;

public class Car {
    private String make;
    private String model;
    private int year;

    public Car(String make, String model, int year) {
        this.make = make;
        this.model = model;
        this.year = year;
    }

    public void displayInfo() {
        System.out.println(year + " " + make + " " + model);
    }

    public static void main(String[] args) {
        Car car1 = new Car("Hyundai", "Elantra", 2013);
        car1.displayInfo();
    }
}

File: Objects_Practice/Car.py
class Car:
    def __init__(self, make, model, year):
        self.make = make
        self.model = model
        self.year = year
    
    def displayInfo(self):
        print(f"{self.year} {self.make} {self.model}")

car1 = Car("Hyundai", "Elantra", 2013)
car1.displayInfo()